### DISCUSSION TITLE
去和去除正文里面偶数页的空白页

### DISCUSSION THREAD

**Discussion by zijuzhang** [whutug/whu-thesis#192] [General]
在正文 文献之后都存在偶数页为空白的现象 如何去除呢

**Comment by note286** [comment]
因为要双面打印，每章都开始于奇数页。

**Reply by zijuzhang** [reply]
上传时不得把这儿去掉呀

**Reply by sikouhjw** [reply]
> 上传时不得把这儿去掉呀

https://github.com/whutug/whu-thesis/issues/117#issuecomment-1097500788

**Comment by sikouhjw** [comment]
https://github.com/whutug/whu-thesis/issues/44#issuecomment-796412253

**Reply by zijuzhang** [reply]
多谢 再咨询一下 文献怎么增加标号 我的参考文献前面没有标号 

![iShot_2023-03-16_18 28 07](https://user-images.githubusercontent.com/14990346/225589483-0427bbee-489c-4687-bf76-92d1c6cdf5be.png)

**Reply by sikouhjw** [reply]
> 多谢 再咨询一下 文献怎么增加标号 我的参考文献前面没有标号

1. 跟主题无关的问题去新开 issue
2. 这是你的认知错误，修正你的认知，硕博要求使用 `author-year` 风格，这是规定
3. 如果你需要的不是 `author-year`，说明你是本科生，你需要的是 `bachelor`，这是你的设置问题

**Reply by earthmoon** [reply]
> > 多谢 再咨询一下 文献怎么增加标号 我的参考文献前面没有标号
> 
> 1. 跟主题无关的问题去新开 issue
> 2. 这是你的认知错误，修正你的认知，硕博要求使用 `author-year` 风格，这是规定
> 3. 如果你需要的不是 `author-year`，说明你是本科生，你需要的是 `bachelor`，这是你的设置问题

我刚去看了规定，武大推荐使用`author-year`版式，但是最后一段说了可以使用顺序编码制，需参考2005版国标。不过顺序编码制不应该和`author-year`混用，他的确不应该在前面加编号。
